### ISSUE TITLE
Not compiling on Windows. 

### ISSUE THREAD

**Issue by ak-agus** [otadrive/OTAdriveESP#5]
hi the library works perfectly on mac but when i go to windos it throws a lot of errors. 

![image](https://user-images.githubusercontent.com/61665853/204652598-8df310a3-c820-43cc-929a-5691ec3d223d.png)

**Comment by mostafahk** [comment]
Hi
Please update your ESP platform to the latest version (v5.2.0 for ESP32, v4.0.1 for ESP8266).

**Comment by ak-agus** [comment]
Hi mostafahk, thanks for your response, but it not the case i have in both computers same versions of all, code, esp platform etc. 
thanks

**Comment by mostafahk** [comment]
can I have your test project in windows? At least the platform.ini file

**Comment by ak-agus** [comment]
yes offcourse, same ini that mac(working code), versions and plarform all the same esp 5.2 

`[platformio]
default_envs = esp32dev
data_dir = src/data

[common]
version = -D BUILD_TAG=v3.4.rc0-Build-20221019
lib_deps = 
	bblanchon/ArduinoJson @ ^6.18.5
	knolleary/PubSubClient @ ^2.8
	paulstoffregen/Time @ ^1.6.1
	arduino-libraries/NTPClient @ ^3.2.1
	adafruit/RTClib @ ^2.1.1
	mykeels/ReadLines @ ^1.0.0 ;LIBRERIA PARA LEER LINEA POR LINEA EN LA SD
	adafruit/Adafruit INA219 @ ^1.2.0
	mikalhart/TinyGPS @ 0.0.0-alpha+sha.db4ef9c97a
	vshymanskyy/TinyGSM @ ^0.11.5
	;otadrive/OTAdrive@1.1.5
	otadrive/OTAdrive @ ~1.1.7

[env:esp32dev] ;esp32dev
platform = espressif32
monitor_filters = esp32_exception_decoder
build_type = debug
framework = arduino
board = esp32dev
;platform_packages =  framework-arduinoespressif32 @ https://github.com/espressif/arduino-esp32.git

;;;;; SECCION DE DEBUG //////////////////////////////////////

;debug_speed = 1000
;upload_protocol = jlink
;monitor_speed = 115200
;debug_tool = jlink
;debug_init_break = tbreak setup

;;;;; ///////////////// FIN SECCION DEBGUG //////////////////
board_build.mcu = esp32
board_build.f_cpu = 240000000L
board_build.flash_mode = qio
board_upload.flash_size = 16MB
board_upload.maximum_size = 16777216

board_build.partitions = large_spiffs_16MB.csv
upload_protocol = esptool
lib_deps = 
	${common.lib_deps}
	plerup/EspSoftwareSerial@^6.16.1
	arduino-libraries/ArduinoHttpClient@^0.4.0
	vshymanskyy/StreamDebugger@^1.0.1
	;robtillaart/SHT2x@^0.2.0
	
build_src_flags = ${common.version}
 ;-DASYNCWEBSERVER_REGEX
build_flags = 
 
 -DPIO_FRAMEWORK_ARDUINO_LWIP_HIGHER_BANDWIDTH 
upload_speed = 921600
monitor_speed = 115200

;upload_port = /dev/cu.SLAB_USBtoUART
;monitor_port =  /dev/cu.SLAB_USBtoUART

upload_port = COM3
monitor_port = COM3
monitor_raw = no
;monitor_rts = 0
;monitor_dtr = 0
`

**Comment by mostafahk** [comment]
It seems there is a conflict when using ```Arduino-libraries/ArduinoHttpClient@^0.4.0``` and the framework HTTPClient lib.
I'm still working on it.

**Comment by ak-agus** [comment]
interesting, but the rare thing is why works on mac and on windows doesn't, same code, same environment

**Comment by ak-agus** [comment]
Hi, any update about this ?

**Comment by skndungu** [comment]
@mostafahk  any update on this since then. 
It seems there is a conflict when using Arduino-libraries/ArduinoHttpClient@^0.4.0 and the framework HTTPClient lib.
I'm still working on it.

**Comment by inoutglobal** [comment]
Same here, impossible to compile on Windows.

[env:esp32dev]
monitor_speed = 115200
platform = espressif32
board = esp32dev
framework = arduino
lib_deps = 
	vshymanskyy/TinyGSM@^0.11.7
	vshymanskyy/StreamDebugger@^1.0.1
	govorox/SSLClient@^1.0.0
	arduino-libraries/ArduinoHttpClient@^0.4.0
	otadrive/OTAdrive@^1.1.13

**Comment by skndungu** [comment]
@mostafahk  any progress on resolving the issue on the conflict, the issue comes from this declaration: 
```
// // MuxChannel 1 for OTA
TinyGsmClient gsm_otadrive_client(modem, 1);
```
without this everything works well but once included, neither POST nor GET gets server response from gprs http requests

**Comment by skndungu** [comment]
> Same here, impossible to compile on Windows.
> 
> [env:esp32dev] monitor_speed = 115200 platform = espressif32 board = esp32dev framework = arduino lib_deps = vshymanskyy/TinyGSM@^0.11.7 vshymanskyy/StreamDebugger@^1.0.1 govorox/SSLClient@^1.0.0 arduino-libraries/ArduinoHttpClient@^0.4.0 otadrive/OTAdrive@^1.1.13

for compiliation just remove 
`otadrive/OTAdrive@^1.1.13`
on your lib deps then download and put the entire OTA lib on the lib folder
